home/tests/test_api.py

```diff
@@ -376,15 +410,17 @@ def test_whatsapp_detail_view_with_button(self):
         homepage.add_child(instance=page)
         page.save_revision().publish()
 
-        response = self.client.get(f"/api/v2/pages/{page.id}/?whatsapp=true&message=1")
+        response = uclient.get(f"/api/v2/pages/{page.id}/?whatsapp=true&message=1")
         content = response.json()
         [button] = content["body"]["text"]["value"]["buttons"]
         button.pop("id")
-        self.assertEqual(
-            button, {"type": "next_message", "value": {"title": "Tell me more"}}
-        )
+        assert button == {"type": "next_message", "value": {"title": "Tell me more"}}
 
-    def test_whatsapp_template(self):
+    def test_whatsapp_template(self, uclient):
+        """
+        FIXME:
+         * Is this actually a template message?
```

Comment:
Yeah I think there's a boolean field here that also needs to be set, `is_whatsapp_template`, as well as `whatsapp_template_name`

Reply:
🤔

Reply:
I'm going to leave this as-is for now.